app/models/health/soap/mass_health.rb

```diff
@@ -48,7 +48,10 @@ def file_list
       file_list = Array.wrap(Hash.from_xml(result.response)&.dig('FileList', 'File')).uniq
       return ::Health::Soap::FileList.new(file_list, self)
     rescue StandardError
-      raise result.response.error_message
+      message = result&.response&.error_message
+      raise message if message.present?
+
+      raise # Don't swallow the original exception if we don't have an error message from the API
```

Comment:
Do you want to raise a string so we know what's going on?

Reply:
`raise` by itself in the context of a handler, should preserve the original exception and call stack, so we can see the source of the original problem.